### PR TITLE
Use bulk update API for shard allocation

### PR DESCRIPTION
**Pull request by chbatey**
On top of #1138

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -9,7 +9,7 @@ val Nightly = sys.env.get("TRAVIS_EVENT_TYPE").contains("cron")
 val Scala211 = "2.11.12"
 val Scala212 = "2.12.10"
 val Scala213 = "2.13.1"
-val akkaVersion26 = "2.6.3"
+val akkaVersion26 = "2.6.6"
```
Comment (ennru) [inline build.sbt:12]: Is this upgrade is required? If not we'd keep 2.6.3.

**Comment by chbatey** [comment]
The new API is not in 2.6.3 but I could check what the min upgrade could be
if that helps? But if we’re upgrading maybe we should just go to 2.6.6

**Comment by ennru** [comment]
Yes, when we update we go all the way.

**Comment by seglo** [comment]
Unrelated build failure: https://github.com/akka/alpakka-kafka/issues/1070

**Pull request merged**